src/components/Template/Template.stories.mdx

```diff
@@ -65,10 +65,10 @@ import { getCategory } from "../../utils/componentCategories";
 ## Table of Contents
 
 - [Accessibility for TemplateAppContainer](#accessibility-for-templateappcontainer)
```

Comment:
Is `Accessibility for TemplateAppContainer` the first item in the TOC for a reason?  Or should it be moved down below `TemplateAppContainer Component Props` and renamed to `TemplateAppContainer Accessibility`?